build/Dockerfile.registry

```diff
@@ -1,15 +0,0 @@
-FROM quay.io/openshift/origin-operator-registry:4.2.0
-
-COPY manifests manifests
-
-# Bundle should include only manifests related to the CSV,
-# drop all additional files.
-RUN find manifests/node-maintenance-operator/*/ -type f ! -name '*_crd.*' ! -name '*.clusterserviceversion.*' -exec rm -f {} \;
-
-# Initialize the database
-RUN initializer --manifests manifests/node-maintenance-operator/ --output bundles.db
-
-# There are multiple binaries in the origin-operator-registry
-# We want the registry-server
-ENTRYPOINT ["registry-server"]
-CMD ["--database", "bundles.db"]
```

Comment:
Does this relate to the old style OLM format thats being deprecated?
Would we have an reason to keep a cleaned up version of this around?

Reply:
This registry image is replaced by the new index image (plus bundle image), and they basically do the same (providing the CSV and CRD manifests), they are just build in different ways.
So I don't see a reason to keep this one.